addons/website_sale/models/sale_order.py

```diff
@@ -21,6 +24,10 @@ class SaleOrder(models.Model):
     )
     cart_quantity = fields.Integer(compute='_compute_cart_info', string='Cart Quantity')
     only_services = fields.Boolean(compute='_compute_cart_info', string='Only Services')
+    is_abandoned_cart = fields.Boolean(compute='_compute_abandoned_cart', string='Abandoned Cart')
```

Comment:
If you store this field (easy) or make a search function (less easy) for it, you can simplify the code in your dashboard and in the wizard